src/style.css

```diff
@@ -1,3 +1,60 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --link: #00acdc;
+}
+
 body {
   margin: 0;
+  background-color: rgb(216, 216, 216);
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+  width: 100%;
+}
+
+.container img {
+  margin: 10px 419px 0 20px;
```

Comment:
The large right margin (419px) on the '.container img' selector seems excessive and could cause layout issues. Verify if such a large margin is necessary, and consider making it smaller or using a different approach to position the image.